homeassistant/components/sensibo/climate.py

```diff
@@ -316,59 +318,35 @@ async def async_set_temperature(self, **kwargs):
             else:
                 return
 
-        with async_timeout.timeout(TIMEOUT):
-            await self._client.async_set_ac_state_property(
-                self._id, "targetTemperature", temperature, self._ac_states
-            )
+        await self._async_set_ac_state_property("targetTemperature", temperature)
 
     async def async_set_fan_mode(self, fan_mode):
         """Set new target fan mode."""
-        with async_timeout.timeout(TIMEOUT):
-            await self._client.async_set_ac_state_property(
-                self._id, "fanLevel", fan_mode, self._ac_states
-            )
+        await self._async_set_ac_state_property("fanLevel", fan_mode)
 
     async def async_set_hvac_mode(self, hvac_mode):
         """Set new target operation mode."""
         if hvac_mode == HVAC_MODE_OFF:
-            with async_timeout.timeout(TIMEOUT):
-                await self._client.async_set_ac_state_property(
-                    self._id, "on", False, self._ac_states
-                )
+            await self._async_set_ac_state_property("on", False)
```

Comment:
So we might get in weird waters right now, but if this failed, the automation should fail.
Just silently continuing might disturb the sequence defined in weird ways.

Reply:
Fixed to raise an exception and updated the PR description